server/apps/impl/bindings.go

```diff
@@ -70,21 +66,26 @@ func (s *service) scanAppBindings(app *apps.App, bindings []*apps.Binding, locPr
 			}
 		}
 		if !allowed {
-			return nil, errors.Errorf("location %s is not granted to app %s", fql, app.Manifest.AppID)
+			// TODO Log this somehow to the app?
+			s.Mattermost.Log.Debug(fmt.Sprintf("location %s is not granted to app %s", fql, app.Manifest.AppID))
```

Comment:
I've been thinking about this, how can we communicate an error message to the app? But who will be reading the error, likely the MM admin? So having it in the MM logs may be the right place.

Reply:
2/5 We should hook in the `Logger` soon enough and send the logs to the App's "admin" channel.

Reply:
The discussion here is "who can fix things". Certain errors may make sense to log to the Mattermost instance (app admin, sysadmin, or whoever we want in the Mattermost instance). But for locations not granted, there is no actionable thing to do from the Mattermost side. The App provider is the one that should ask for more permissions or stop trying to register that location.

Also, this particular log is quite spammy whatever we do (as soon as one location is misconfigured, any time any user navigates to a channel, it will create a new log message).